pkg/build/controller/build/overrides/overrides.go

```diff
@@ -32,22 +32,28 @@ func (b BuildOverrides) ApplyOverrides(pod *corev1.Pod) error {
 
 	klog.V(4).Infof("Applying overrides to build %s/%s", build.Namespace, build.Name)
 
-	if b.Config.ForcePull {
+	if b.Config.ForcePull != nil {
```

Comment:
could we change this to `if b.Config.ForcePull != nil && *b.Config.ForcePull` and then leave most of what is below the same?

Reply:
duh, nevermind ... if they switch back to false ... see @coreydaley 's https://github.com/openshift/openshift-controller-manager/pull/123#issuecomment-655558903

feel free to resolve this one @coreydaley